lib/streamlit/elements/doc_string.py

```diff
@@ -131,8 +131,4 @@ def _unwrap_decorated_func(f):
         except AttributeError:
             pass
 
-    # Falling back to original function. Which is fine in Python 3 for
-    # functions, even if they were decorated with functools.wrap.  No such luck
-    # for Python 2, but it's unlikely we'll get to this part of the code
-    # anyway.
```

Comment:
It seems potentially worth keeping this comment, minus the Python 2/3 distinctions, so that we're still explicitly calling out that, in the case of an exception, we are ok falling through to the original function.